protocol/messenger_communities.go

```diff
@@ -1736,7 +1750,14 @@ func (m *Messenger) LeaveCommunity(communityID types.HexBytes) (*MessengerRespon
 	if err != nil {
 		return nil, err
 	}
+	mr.AddActivityCenterNotifications(notifications)
```

Comment:
not sure I understand this code, why have we changed delete/dismiss, and why are we repulling and readding all notifications?

Reply:
It was an attempt to delete all of the notifications related to the community on leaving.
Anyways, I am drafting this now as we decided to delay working on this for now.

Reply:
:+1
I think we should start testing the only change we have made before, and take it from there